pages/admin/invite/userAccountPage/[id].tsx

```diff
@@ -0,0 +1,156 @@
+import DashboardLayout from "components/DashboardLayout";
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { DeleteUserDTO } from "pages/api/admin/users/delete";
+import Combobox from "components/Combobox";
+import { User } from "@prisma/client";
+import Button from "components/Button";
+import { UpdateUserDTO } from "pages/api/admin/users/update";
+import { UserRoleType, IUser } from "interfaces";
+
+const UserAccountPage: React.FunctionComponent<UserRequest> = ({
+  onDelete,
+  onAccept,
+}) => {
+  const router = useRouter();
+  const [selectedPlayers, setSelectedPlayers] = useState<User[]>([]);
+  const [user, setUser] = useState<IUser>();
+  const { id } = router.query;
+  useEffect(() => {
+    const getUser = async (): Promise<void> => {
+      const response = await fetch(`/api/admin/users/${id}`, {
+        method: "GET",
+        headers: { "content-type": "application/json" },
+        redirect: "follow",
+      });
+      const data = await response.json();
+      setUser(data.user);
+    };
+    getUser();
+  }, [id]);
+
+  const showCombobox = (): string => {
+    return ([
+      UserRoleType.Mentor,
+      UserRoleType.Parent,
+      UserRoleType.Donor,
+    ] as UserRoleType[]).includes(user?.defaultRole.type as UserRoleType)
+      ? ""
+      : "hidden";
+  };
+  const deleteUser = async (): Promise<void> => {
```

Comment:
An error is thrown when you click delete on the specific user page. Seems to be something with your `id`